rdkit/Chem/MolStandardize/tautomer.py

```diff
@@ -9,11 +9,11 @@
 :license: MIT, see LICENSE file for more details.
```

Comment:
I don't think you need to make any edits at all to this file for this PR

Reply:
I've manually put the changes back. To be honest, not sure how they came about—suspect it was Pycharm? Will keep an eye out for any like these in the future.